src/components/addTask.js

```diff
@@ -15,17 +15,18 @@ function AddTask({ user }) {
   };
   toast.configure();
   const success = () =>
-    toast.info("Successfuly Added Event: " + eventName.value);
+    toast.info("Successfuly Added Task: " + eventName.value);
+  const failure = () =>
+    toast.warn("Failed to Add Task " + eventName.value);
 
   let maxDate = new Date(2022, 12, 30);

```

Comment:
Just saw this, but maxDate should probably change based on the current date, else the date picker would break in December of next year.